src/java/com/wolfssl/provider/jsse/WolfSSLEngine.java

```diff
@@ -1091,6 +1091,9 @@ else if (hs == SSLEngineResult.HandshakeStatus.NEED_WRAP &&
                     (err != WolfSSL.SSL_ERROR_WANT_READ) &&
                     (err != WolfSSL.SSL_ERROR_WANT_WRITE)) {
                     if (err == WolfSSL.UNKNOWN_ALPN_PROTOCOL_NAME_E) {
+                        /* Native wolfSSL could not negotiate an ALPN protocheckout col,
```

Comment:
I'm having a hard time understanding what "protocheckout col" means.  I thin we can just say "Native wolfSSL could not negotiate a common ALPN protocol"

Reply:
Looks like I didn't switch windows when typing git checkout in the terminal! Thanks!